students/ghassan/lesson02/fizzbuzz.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+for i in range(1, 100):
+    if not i%3 and not i%5:
+            print('FizzBuzz')
+    elif not i%3:
+            print('Fizz')
```

Comment:
check the spacing on your editor, you want exactly 4 spaces indentation for python